MCprep_addon/optimize_scene.py

```diff
@@ -92,7 +102,10 @@ class MCPrep_OT_optimize_scene(bpy.types.Operator):
 
 	def execute(self, context):
 		global MAX_BOUNCES
```

Comment:
This `global MAX_BOUNCES` line too can be removed actually, not needed unless you are assigning to it (which isn't advised, since `MAX_BOUNCES` should be thought of as a constant)